tests/BUILD.txt

```diff
@@ -0,0 +1,7 @@
+Building aarch64-w64-mingw32 tests on Linux or WSL
+
+cmake -S . -B build
+cmake --build build
+
+it generates Arm64 Windows executable
+build/aarch64_mingw_tests
```

Comment:
So far the repository is using `-` as word delimiter in file names. This adds inconsistency. The same applies for file names of all the tests in this folder and for git branch names.

Reply:
Done